vm/src/frame.rs

```diff
@@ -163,6 +169,7 @@ impl Frame {
     /// Execute a single instruction.
     #[allow(clippy::cognitive_complexity)]
     fn execute_instruction(&self, vm: &VirtualMachine) -> FrameResult {
+        check_signals(vm);
```

Comment:
Add here something like this:

`#[cfg(not(target_arch = "wasm32"))]`

Reply:
Done